misc/theme-cicada/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		1.12
+PLUGIN_VERSION=		1.12_2
```

Comment:
to do this, use PLUGIN_REVISION

Reply:
better clear it, this looks more like a feature anyway (PLUGIN_VERSION=1.13 and delete PLUGIN_REVISION)

Reply:
```suggestion
PLUGIN_VERSION=		1.13
```

also possible directly as a delta :)